README.adoc

```diff
@@ -0,0 +1,83 @@
+[[minishift]]
+= Minishift
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+toc::[]
+
+[[welcome-to-minishift]]
+== Welcome to Minishift!
+
+Minishift is a tool that helps you run OpenShift locally by running a
+single-node OpenShift cluster inside a VM. You can try out OpenShift or
+develop with it, day-to-day, on your local host.
+
+Minishift uses https://github.com/docker/machine/tree/master/libmachine[libmachine] for
+provisioning VMs, and https://github.com/openshift/origin[OpenShift Origin] for running the cluster.
+
+https://travis-ci.org/minishift/minishift[image:https://secure.travis-ci.org/minishift/minishift.png[Build Status]]
+https://ci.appveyor.com/project/hferentschik/minishift-o61ou/branch/master[image:https://ci.appveyor.com/api/projects/status/6wyv1cpd588cm4ce/branch/master?svg=true[Build status]]
+https://circleci.com/gh/minishift/minishift/tree/master[image:https://circleci.com/gh/minishift/minishift/tree/master.svg?style=svg[Build status]]
+https://ci.centos.org/job/minishift/[image:https://ci.centos.org/buildStatus/icon?job=minishift[Build Status]]
+
+'''''
+
+[[getting-started]]
+== Getting started
+
+To download the latest binary and review release notes, see
+the https://github.com/minishift/minishift/releases[Minishift releases] page.
+
+Minishift requires a hypervisor to start the virtual machine containing
+OpenShift. Make sure that the hypervisor of your choice is installed and
+enabled on your system before you install Minishift.
+
+For detailed installation instructions for Minishift and the required dependencies, see
+the https://docs.openshift.org/latest/minishift/getting-started/index.html[Getting started] documentation.
+
+[[documentation]]
+== Documentation
+
+Minishift documentation is published as a part of the OpenShift Origin documentation librarry.
+Check out the latest official Minishift documentation for information about getting started,
```

Comment:
'latest official' :) not sure those words are needed here?
Docs in GitHub are not unofficial, nor would they be stale, in fact these would be more current than the docs on the site right?